tests/chart_tests/test_nats_jetstream.py

```diff
@@ -13,6 +13,7 @@ def test_nats_statefulset_with_jetstream(self, kube_version):
         """Test that nats statefulset is good with defaults."""
         values = {
             "global": {"nats": {"jetStream": {"enabled": True}}},
+            "nats": {"nats": {"createJetStreamJob": True}},
```

Comment:
this is a default test cases we are ensuring templates are rendered or not , so its disabled by default it should just asserted with len(docs)

Reply:
Ok, so the assert len(docs) should be 3, but then there is a condition in the next lines, should that be removed?:
`assert "jetstream" in nats_cm
  assert {"runAsUser": 1000, "runAsNonRoot": True} == docs[2]["spec"]["template"]["spec"]["containers"][0]["securityContext"]`

Reply:
done.